data/db/src/iosMain/kotlin/io/github/droidkaigi/feeder/data/AppleSettings.kt

```diff
@@ -0,0 +1,197 @@
+package io.github.droidkaigi.feeder.data
+
+import com.russhwolf.settings.ExperimentalSettingsApi
+import com.russhwolf.settings.ObservableSettings
+import com.russhwolf.settings.Settings
+import com.russhwolf.settings.SettingsListener
+import kotlin.native.concurrent.AtomicReference
+import kotlin.native.concurrent.freeze
+import platform.Foundation.NSNotification
+import platform.Foundation.NSNotificationCenter
+import platform.Foundation.NSUserDefaults
+import platform.Foundation.NSUserDefaultsDidChangeNotification
+import platform.darwin.NSObjectProtocol
+
+/*
+ Reference:
+ https://github.com/russhwolf/multiplatform-settings/blob/master/multiplatform-settings/src/appleMain/kotlin/com/russhwolf/settings/AppleSettings.kt
```

Comment:
I have a question! Why did you need a copy of the code? 🙋

Reply:
I may have made a mistake...
Thank you so much🙏
I will investigate‼️

Maybe I shouldn't have added the flag to `gradle.properties` 🤔  (#298)

Reply:
Thank you for the investigation! Kotlin MPP requires a lot of trial and error, so let's try it 👍

Reply:
I'm guessing the problem is that `multiplatform-settings` doesn't have the` kotlin.mpp.enableGranularSourceSetsMetadata` flag.
Syntax highlighting doesn't work even with #298.
I tried a workaround to add ios target. 9e2cd3a